ext/opentelemetry-ext-http-requests/setup.cfg

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+[metadata]
+name = opentelemetry-ext-http-requests
```

Comment:
why "http-requests"? It feels to me like calling this "opentelemetry-ext-requests" might be clearer.

For example, I imagine flask integration being along the lines of "opentelemetry-ext-flask" rather than "opentelemetry-ext-httpserver-requests".

Trying to put how I feel into words, I think one who is looking for an integration will find it via the package name alone, and the additional categorization is unneeded.

Reply:
@toumorokoshi I kind of agree with you, this is what I did in OpenCensus https://github.com/census-instrumentation/opencensus-python/tree/master/contrib.

Reply:
I agree but it seems that pylint gets confused when naming it `requests`. Please look at the commit history of this PR.